src/hdr_histogram.c

```diff
@@ -337,17 +353,20 @@ int hdr_init(
         return r;
     }
 
+    hdr_mutex_lock(&hdrMutex);
```

Comment:
Note that until this function returns the caller cannot get access to the newly-created histogram object - and hence there's no possibility another thread could be manipulating it. 

As such I don't believe you need to acquire the mutex in this function.

Reply:
Thanks Dave good point (bit of a derp here)

Reply:
Done